src/Context.js

```diff
@@ -0,0 +1,11 @@
+import React, { createContext, useState } from 'react';
+
+export const TheContext = createContext();
```

Comment:
there is no need for The before context and provider. It can simply be Context and StateProvider. This would make it easier to import